st2tests/integration/orquesta/base.py

```diff
@@ -32,6 +32,9 @@
     action_constants.LIVEACTION_STATUS_RUNNING
 ]
 
+DEFAULT_WAIT_FIXED = 3000
```

Comment:
Why not just change this to 1500 default so you don't have to override it?

Reply:
Sorry, this should actually default to ``500``, not ``3000``.

I was testing many things and accidentally added a wrong change.

We should default to the lowest value possible which works for most of the tests and increase it for tests which need a higher value (like the ones which explicitly set it to 1500).

Will again verify ``500`` works correctly and push the change.

Reply:
Ok.  I'll pull down the changes later this PM and make a run myself.  I still need to understand the changes you made to wait state method.  I'll merge this PM if all is ok.